### ISSUE TITLE
[Help Wanted] Did pricing just change dramatically?

### ISSUE THREAD

**Issue by Paul-5** [standardnotes/forum#1454]
About a week ago, I purchased Standard Notes Extensions for around $149 for five years... i.e. $30 per year. This is equivalent to the Pro Plan which is now advertised for $119/year. Has there been a dramatic jump in pricing or has something changed to justify the price difference?

I'm concerned that if I invest time and effort into Standard Notes now that it will cost over $100/year when I renew, and unfortunately it doesn't compare favorably with its competition at that price point. If this is the case I'm better off moving to another product now.

**Comment by Mooie** [comment]
I have similar concerns.  I subscribed to a five year plan, for what I felt was a very reasonable price.  But, at over 100 dollars a year, I'm unlikely to stay a customer.  I like the Standard Notes philosophy and product.  And, I understand the need to generate income for the developers.  Everyone has to earn a living and cover the costs.  Perhaps, for this level of security and cross-platform support, that's what you have to pay?  If so, then it's too rich for me, and I'll go elsewhere.  That's fair enough, I think.  However, it would have been better form to warn folks buying longer term subscriptions that a large price rise was likely at some future point.  That way we could have made more of an informed choice.  But, I suspect, the price increase was more a case of having to respond to an emerging reality, rather than having planned it from the start.  That said, an ongoing discount for early adopters would be a nice gesture, and may be something that the SN team might consider?

**Comment by moughxyz** [comment]
Sorry about the confusion. We sent out a mass user email explaining the change but only retrospectively determined it was not sent to current subscribers (as is the case for marketing emails, which we send out probably once a year).

To summarize, existing subscribers lock in their rate and are perpetually grandfathered in. So long as you don't cancel your subscription, it will renew at the price you signed up with. Subscription price increases always only apply to new subscribers.

**Comment by thegrizz** [comment]
yeah 80/year just to "unlock" the checkmark box or to have it be able to understand your doing a number list is not reasonable

**Comment by DrownedStarfish** [comment]
**> So long as you don't cancel your subscription, it will renew at the price you signed up with.**

Only one problem: I paid then with PayPal, and that recurring PayPal link is no longer active.  What happens now?  SN will try to take the next payment, and the payment won't get paid.

Thanks.
DS

**Comment by stefan-yas** [comment]
@DrownedStarfish I'd suggest reaching out at help@standardnotes.com with the email address you're using for Standard Notes, as well as your PayPal email and my team mate will shed more light on the situation with your account.

**Comment by DrownedStarfish** [comment]
> @DrownedStarfish I'd suggest reaching out at [help@standardnotes.com](mailto:help@standardnotes.com) with the email address you're using for Standard Notes, as well as your PayPal email

Request sent to help@standardnotes.com, as advised.

Thanks.
DS

**Comment by CubeOfCheese** [comment]
> yeah 80/year just to "unlock" the checkmark box or to have it be able to understand your doing a number list is not reasonable

It's now 59/year to unlock checkboxes and lists and that still feels way too high. I'd gladly pay a starter fee of like 5-10/year for those basic features but definitely not 59
